packages/synapse-react-client/test/lib/containers/DownloadLoginModal.test.tsx

```diff
@@ -33,15 +33,15 @@ describe('DownloadLoginModal tests', () => {
 
   it('Has cancel button', async () => {
     renderComponent()
-    const cancelButton = screen.getByRole('button', { name: 'CANCEL' })
+    const cancelButton = screen.getByRole('button', { name: /cancel/i })
```

Comment:
Use case insensitive search so test doesn't break if button capitalization changes

Reply:
Could instead use a constant (referenced in both the component and test)